docs/source/guides/index.mdx

```diff
@@ -122,13 +122,14 @@ If you register a persisted query with a specific client name instead of `null`,
 
 Use the `--header` option when running the MCP Server to pass the header to the router. The default name of the header expected by the router is `apollographql-client-name`. To use a different header name, configure `telemetry.apollo.client_name_header` in router YAML configuration.
 
-### From schema introspection
+## Introspection tools
 
-For use cases where not all operations can be pre-defined, Apollo MCP Server supports tool creation based on introspection of the graph schema. This allows AI agents to explore a graph and execute operations dynamically.
+In addition to defining specific tools for pre-defined GraphQL operations, the Apollo MCP Server supports introspection tools that allow AI agents to explore the graph schema and execute operations dynamically.
```

Comment:
I see your correct `Apollo MCP Server` to `the Apollo MCP Server`. If that's intentional, when should we use Apollo MCP Server instead of the Apollo MCP Server? This can be tricky for non-native English speakers like me 😂, so I need a clear guideline to keep our documentation consistent.

Reply:
I'm still curious about this, so I'll bring it up in the inbox-docs channel. Thanks!